src/Repository.java

```diff
@@ -83,5 +85,59 @@ public static Repository getInstance() {
         }
         return result;
     }
+    
+    public void getSongs() throws SQLException {
+    	statement = connection.createStatement();
+    	
+    	if(statement.execute("SELECT * FROM song")) {
+    		ResultSet rs = statement.getResultSet();
+    		while(rs.next()) {
+    			// put songs from library into the jTable somehow
+    		}
+    	}
+    }
+
+    
+    public void addSong(String fn) throws SQLException {
+        String filePath = fn;
+        
+        try {
+            Connection conn = DriverManager.getConnection(DB_URL, DB_USER, DB_PASSWD);
```

Comment:
There's an existing `connection` object, we can just reuse it instead of creating new one here. Database connection object is expensive.